app/src/main/java/de/westnordost/streetcomplete/quests/opening_hours/OpeningHoursTagParser.kt

```diff
@@ -0,0 +1,463 @@
+package de.westnordost.streetcomplete.quests.opening_hours
+
+import ch.poole.openinghoursparser.*
+import de.westnordost.streetcomplete.quests.opening_hours.adapter.OpeningMonthsRow
+import de.westnordost.streetcomplete.quests.opening_hours.adapter.OpeningWeekdaysRow
+import de.westnordost.streetcomplete.quests.opening_hours.model.CircularSection
+import de.westnordost.streetcomplete.quests.opening_hours.model.OpeningMonths
+import de.westnordost.streetcomplete.quests.opening_hours.model.TimeRange
+import de.westnordost.streetcomplete.quests.opening_hours.model.Weekdays
+import java.io.ByteArrayInputStream
+
+object OpeningHoursTagParser {
+    // returns null for values that are invalid or not representable in
+    // StreetComplete opening hours edit widget
+    // otherwise returns data structure that can be directly used to
+    // initialize this editing widget
```

Comment:
Documentation comments should be written like this `/** */`

Reply:
Thanks, should be now fixed here and elsewhere.